src/app/ui/palette_popup.cpp

```diff
@@ -38,7 +38,7 @@ PalettePopup::PalettePopup()
 
   m_popup->loadPal()->Click.connect(base::Bind<void>(&PalettePopup::onLoadPal, this));
   m_popup->openFolder()->Click.connect(base::Bind<void>(&PalettePopup::onOpenFolder, this));
-
+  m_paletteListBox.DoubleClickItem.connect(base::Bind<void>(&PalettePopup::onLoadPal, this));
```

Comment:
I'd move this right before/after row 44 (`m_paletteListBox.PalChange.connect(&PalettePopup::onPalChange, this);`)